spec/controllers/educations_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe EducationsController, type: :controller do
+
+  describe 'GET #index' do
+    # it "populates an array of all educations" do
+    #   monte = create(:education, school: 'Monte')
+    #   ssori = create(:education, school: 'ssori')
+    #   get :index
+    #   expect(assigns(:education)).to match_array([monte, ssori])
+    # end
```

Comment:
I agree. Definitely. I am working on these controller tests now. 

For future github protocol, should I not do a pull request until all items on the card are addressed? My thinking was to make a pull request for what I had, in case anyone wanted to look at it as an example. I haven't moved my trello card at this point because I am still working on the controller tests (which are a little more intensive).